tools/clang/lib/Sema/SemaHLSL.cpp

```diff
@@ -4809,11 +4996,17 @@ class HLSLExternalSource : public ExternalSemaSource {
           m_usedIntrinsics.insert(UsedIntrinsic(pIntrinsic, functionArgTypes));
       bool insertedNewValue = insertResult.second;
       if (insertedNewValue) {
+        NamespaceDecl *nsDecl = m_hlslNSDecl;
+        if (isVkNamespace)
+          nsDecl = m_vkNSDecl;
+        else if (isDxNamespace) {
+          nsDecl = m_dxNSDecl;
+        }
```

Comment:
https://llvm.org/docs/CodingStandards.html#don-t-use-braces-on-simple-single-statement-bodies-of-if-else-loop-statements
```suggestion
        else if (isDxNamespace)
          nsDecl = m_dxNSDecl;
```

Reply:
Applied in [ba11d4f](https://github.com/microsoft/DirectXShaderCompiler/pull/7097/commits/ba11d4f3ffde25c5945852099caf9e983b2deb58)